README.md

```diff
@@ -65,6 +65,9 @@ Using ImGui-SFML in your code
     - Poll and process events:
 
         ```cpp
+        // be sure to call this if you want to support multiple windows
```

Comment:
Change "want to support multiple windows" with "have multiple windows"

Reply:
Also some things I'm not sure about
1. If I create new window, what should I do?
2. If I close the window, should I call something to not leak resources?
3. If I switch the current window, what function should be called to set it as "current"?

Reply:
I changed some parts of the README. Take a look at it and please give some feedback :)